R/outputs.R

```diff
@@ -1322,10 +1326,21 @@ get_period_metadata <- function(calendar_quarters) {
 #' @return The read data
 #' @export
 read_hintr_output <- function(path) {
-  if (tolower(tools::file_ext(path)) == "qs") {
+  type <- tolower(tools::file_ext(path))
+  if (type == "qs") {
     qs::qread(path)
+  } else if (type == "duckdb") {
+    read_duckdb(path)
```

Comment:
with three options here now, consider explicitly adding `type == "rds"` and then a final `else stop(...)` branch